ambry-named-mysql/src/main/resources/NamedBlobsSchema.ddl

```diff
@@ -20,3 +20,29 @@ CREATE INDEX named_blobs_et ON named_blobs(expires_ts);
 
 /* Reverse Lookup Index */
 CREATE INDEX named_blobs_id ON named_blobs(blob_id);
+
+/* New Table */
+CREATE TABLE IF NOT EXISTS named_blobs_v2 (
+    account_id int NOT NULL,
+    container_id int NOT NULL,
+    blob_name varchar(191) NOT NULL,
+    blob_id varbinary(50) NOT NULL,
+    deleted_ts datetime(6) DEFAULT NULL,
+    expires_ts datetime(6) DEFAULT NULL,
```

Comment:
As we discussed, are we suppose to remove the expires_ts and use deleted_ts only?

Reply:
Yes, just did that in latest commit [6f3c303](https://github.com/linkedin/ambry/pull/2143/commits/6f3c3031ab43eda4bd25a40f9968884870eced8f)